### ISSUE TITLE
Correct sed for MAC

### ISSUE THREAD

**Issue by sbocconi** [dyne/Writedown#5]
The `render` script gives a small error on Mac that could be fixed with the following:
```
Stefano-MacBook-Pro:writedown SB$ git diff origin/master
diff --git a/render b/render
index 1ede413..2758708 100755
--- a/render
+++ b/render
@@ -110,8 +110,14 @@ correct_extended_chars() {
     extended=`perl -ane '{ if(m/[[:^ascii:]]/) { print  } }' $dest`
        if [[ ${#extended} -gt 0 ]]; then
                errl=`print $extended | wc -l`
-               sed -i "s/‘/'/g; s/’/'/g; s/…/.../g" $dest
-               sed -i 's/“/"/g; s/”/"/g; s/¨/"/g' $dest
+               if [[ "$(uname) " -eq "Darwin " ]]
+               then
+                       sed -i '' "s/‘/'/g; s/’/'/g; s/…/.../g" $dest
+                       sed -i '' 's/“/"/g; s/”/"/g; s/¨/"/g' $dest
+               else
+                       sed -i'' "s/‘/'/g; s/’/'/g; s/…/.../g" $dest
+                       sed -i'' 's/“/"/g; s/”/"/g; s/¨/"/g' $dest
+               fi
                corrected=`perl -ane '{ if(m/[[:^ascii:]]/) { print  } }' $dest`
                corrl=`print $corrected | wc -l`
                print
```

**Comment by jaromil** [comment]
Hi Stefano!

would the usage of the long-option version of `-i` help? like `--in-place=''`?

Thanks. I have no mac to try it on.

**Comment by sbocconi** [comment]
Unfortunately the Mac seems not to support long options.
sed -i'' works on Mac but not on Linux, and sed -i '' (with a space) the other way around.

**Comment by jaromil** [comment]
Wow! this must be the case also for other commands, as I suspect is a difference between BSD and GNU implementations of getopt...

**Comment by jaromil** [comment]
fixed in 8cbd1467616a26fa8b672065211dbfc492e138e2
